Assets/Global/Scripts/Player/Tails/BaseTail.cs

```diff
@@ -0,0 +1,14 @@
+
+using UnityEngine;
+using System.Collections.Generic;
+
+[CreateAssetMenu(fileName = "BaseTail", menuName = "BaseTail")]
+public class BaseTail : ScriptableObject
+{
+    public List<GameObject> groundCombo = new();
```

Comment:
can this be anything else, or does it need to be GameObject due to it being prefabs?